solana-ibc-story.md

```diff
@@ -598,75 +599,75 @@ mod global_alloc {
 use global_alloc::{global_verifier, set_global_verifier};
 ```
 
-Since Solana runtime is single-threaded no consideration for
-thread-safety is necessary.  Paradoxically this may make things more
+Since the Solana runtime is single-threaded, no consideration for
+thread-safety is necessary.  Paradoxically, this may make things more
 complicated by forcing another version of the code for non-Solana
 builds (e.g. tests).  Another aspect which makes things interesting is
 Solana’s lack of support for atomic operations (e.g. [`AtomicPtr`
 type](https://doc.rust-lang.org/core/sync/atomic/struct.AtomicPtr.html)).
-Mere existence of them, even if they are never executed, causes the
+The mere existence of them on Solana, even if they are never executed, causes the
```

Comment:
‘on Solana’ here is wrong I think. The point is that the instructions never appear on Solana. They are present in the smart contract and that’s what causes the smart contract upload to fail.  Perhaps
```suggestion
The mere existence of them in the smart contract, even if they are dead code, causes the
```